src/hb-ot-color-sbix-table.hh

```diff
@@ -275,6 +337,70 @@ struct sbix
 			  strikes.sanitize (c, this)));
   }
 
+  bool add_strike (hb_subset_context_t *c,
+                   const void *dst_base,
+                   LOffsetTo<SBIXStrike>* o,
+                   unsigned int i) const {
+    *o = 0;
+    if (strikes[i].is_null ())
+      return false;
+
+    auto *s = c->serializer;
+
+    s->push ();
+
+    return (this+strikes[i]).subset (c);
+  }
+
+  bool serialize_strike_offsets (hb_subset_context_t *c,
+                                 const void *dst_base) const
+  {
+    TRACE_SERIALIZE (this);
+
+    auto *out = c->serializer->start_embed<LOffsetLArrayOf<SBIXStrike>> ();
+    if (unlikely (!out)) return_trace (false);
+    if (unlikely (!c->serializer->allocate_size<HBUINT32> (HBUINT32::static_size))) return_trace (false);
```

Comment:
I believe you should be able to use serializer->extend_min(out) which should extend the buffer enough to fit the length parameter.

Reply:
Done.